config/initializers/assets.rb

```diff
@@ -4,4 +4,5 @@
   miq_novnc.js
   spice-html5.js spiceHTML5/spicearraybuffer.js
   codemirror/modes/*.js codemirror/themes/*.css
+  container_topology.css
```

Comment:
I found out the hard way that this is needed. Otherwise showing the topology fails with a message to add it exactly here. So I'll bring it back for a later resolution together with @abonas